builtin/game/item_entity.lua

```diff
@@ -220,7 +221,8 @@ core.register_entity(":__builtin:item", {
 						physical = false
 					})
 				else
-					self.object:set_acceleration({x = -v.x, y = -10, z = -v.z})
+					local slip_factor = 4.0 / (slippery + 4)
+					self.object:set_acceleration({x = -v.x * slip_factor, y = -10, z = -v.z * slip_factor})
```

Comment:
`movement_gravity = 9.81` yes best to match the setting, i'll 'get' the value from settings.